specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -227,7 +227,7 @@
               },
               "Operation-Location": {
                 "type": "string",
-                "description": "New URL to check for the results of the long running process. `api-version` is not included in the URL."
+                "description": "A URL used to check the status or results of the request when it is a [long-running operation](https://aka.ms/am-creator-lrt-v2). `api-version` is not included in the URL."
```

Comment:
Removed link.

Is there anything I should add to the how-to article to clarify?

Reply:
Our API design fully follows the [LRO guideline](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#put-with-additional-long-running-processing) by Azure API review board. But I think we shouldn't include this link in this swagger because the link can be changed, and the content may even change later. Let me read our description again and see if we can add more instruction to the user. But overall, I think the APIs are straightforward and shouldn't be hard to understand.

Reply:
I agree that the swagger docs are fine, I was referring to the how-to article in the user docs. I'm leaning towards not adding anything for this, but wanted your opinion.